Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -788,29 +816,34 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
         /// <summary>
         /// The Input Event handlers receive Hand Ray events.
         /// </summary>
-        public void OnInputDown(InputEventData eventData)
+        public void OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            SetAffordancesActive(false);
             EndTouch(eventData.SourceId);
-            SetHandDataFromController(eventData.InputSource.Pointers[0].Controller, false);
+            SetHandDataFromController(eventData.Pointer.Controller, eventData.Pointer,  false);
             eventData.Use();
         }
-        public void OnInputUp(InputEventData eventData)
+        public void OnPointerUp(MixedRealityPointerEventData eventData)
         {
             EndTouch(eventData.SourceId);
             eventData.Use();
-        }
-        public void OnPositionInputChanged(InputEventData<Vector2> eventData) { }
-        public void OnInputPressed(InputEventData<float> eventData) { }
+        }    
         #endregion IMixedRealityInputHandler Methods
 
-
         #region IMixedRealitySourceStateHandler Methods
-        public void OnSourceDetected(SourceStateEventData eventData) { }
         public void OnSourceLost(SourceStateEventData eventData)
         {
             EndTouch(eventData.SourceId);
             eventData.Use();
         }
         #endregion IMixedRealitySourceStateHandler Methods
+
+        #region Unused Methods
+        public void OnPositionInputChanged(InputEventData<Vector2> eventData) { }
```

Comment:
I think you no longer need OnPositionInputChanged, looks like no interface implements this any more.

Reply:
oh! youre right!

Reply:
removing OnPositionInputChanged & OnInputPressed